usr.sbin/bsdinstall/scripts/bootconfig

```diff
@@ -132,6 +132,9 @@ if [ -n "$(awk '{if ($2=="/boot/efi") printf("%s\n",$1);}' $PATH_FSTAB)" ]; then
 	#
 	if [ ! -f "${mntpt}/${BOOTNAME}" ]; then
 		cp "$BSDINSTALL_CHROOT/boot/loader.efi" "${mntpt}/${BOOTNAME}"
+		if [ -e "$BSDINSTALL_CHROOT/boot/loader-ia32.efi" ]; then
+			cp "$BSDINSTALL_CHROOT/boot/loader-ia32.efi" "${mntpt}/${BOOTDIR}/bootia32.efi"
```

Comment:
Should /efi/freebsd/loader.efi not be the thing your system needs to boot? Or at least have a separate /efi/freebsd/loader(-ia)32.efi? Otherwise you're only using the removable media boot file, which is bad practice.

Reply:
The problem here is that `efibootmgr` relies on `efirt`, but since the ia32 loader doesn't provide the kernel with the EFI system table, we can't use it.
I disabled it mainly because the efirt functions expect to be run in 32-bit protected mode.

Reply:
I realise, but it's confusing to have a single FreeBSD-specific file that's not the right thing, plus if something tramples on the removable media path then you've got no copy on your disk to manually load.

Reply:
That makes sense